pkg/kyma/cmd/install/cmd.go

```diff
@@ -228,7 +230,7 @@ func (cmd *command) configureHelm() error {
 
 	data, ok := cfg["data"].(map[interface{}]interface{})
 	if !ok {
-		return fmt.Errorf("unable to get data from helm secret")
+		return fmt.Errorf("Unable to get data from helm secret")
```

Comment:
I think It is better to keep log messages in the lower case all the time, since it is easier to find the specific message on production, in the stackdriver for example.